src/vs/workbench/contrib/remote/browser/remoteIndicator.ts

```diff
@@ -112,6 +114,10 @@ export class RemoteStatusIndicator extends Disposable implements IWorkbenchContr
 					category,
 					title: { value: nls.localize('remote.showMenu', "Show Remote Menu"), original: 'Show Remote Menu' },
 					f1: true,
+					keybinding: {
+						weight: KeybindingWeight.WorkbenchContrib,
+						primary: KeyMod.Shift | KeyCode.KeyR,
```

Comment:
Are you missing a modifier?

Reply:
Meaning can I still type `R`?

Reply:
And if this won't apply when an editor is focused, I don't think we have other keybindings that are only based on shift?

Reply:
Doh, thanks!